verify-loader

```diff
@@ -116,6 +124,53 @@ def print_stats(invocid, ctx, payload):
                     ctx.count, ctx.skips, ctx.duplicates)
             sys.stdout.flush()
 
+# this returns a dict - if the line is from a loader message, it
+# will contain the following fields:
+# msg - the message from loader
+# line - the full line read, which may include metadata
+# msglen - the length of msg
+# linelen - the length of line
+# invocid - the unique id of the loader run - stripped
+# seq - the sequence number of the message as an int
+# payload - the message payload
+def get_loader_record(line):
+    retval = {}
+    trimpayload = False
+    if line.startswith('loader seq - '):
+        msg = line
+    else:
+        idx = line.find('@cee:{')
+        if idx >= 0:
+            msg = json.loads(line[idx+5:])['message']
+        else:
+            idx = line.find('loader seq - ')
+            if idx >= 0:
+                msg = line[idx:]
+                # payload may have extra metadata at the end
+                trimpayload = True
+            else:
+                msg = None
+    if msg:
+        try:
+            _, invocid, seqval, payload = msg.split('-', 4)
+            seq = int(seqval)
+            msglen = len(msg)
+            if trimpayload:
+                pm = payloadre.match(payload)
+                if pm:
+                    msglen -= len(payload)
+                    payload = pm.group(1)
+                    msglen += len(payload)
```

Comment:
Can you leave `msglen` to be the original length of the line, and add a record entry for payload length?

This will let us see the total bandwidth verify-loader is operating at, and we can then display the payload bandwidth as a sub-set of that.

Reply:
`line` is the original line, and `linelen` is the length of that line.
total bytes read == sum(`linelen`)
`msg` is what was written by `loader`, so
total bytes read that were written by `loader` == sum(`msglen`)